### ISSUE TITLE
Confirm first party domain blocking (#14134) off in Release Channel build

### ISSUE THREAD

**Issue by LaurenWags** [brave/brave-browser#15149]
Placeholder to confirm https://github.com/brave/brave-browser/issues/14134 is off by default on Release Channel builds.

Example:

Visit one of the sites listed in https://github.com/brave/brave-browser/issues/14134#issuecomment-800624427 such as:
* `1-1ads.com`
* `actionsplash.com`
* `pub.chez.com`

Confirm that the site loads as expected, this means that first-party domain blocking is off by default. This is the expected outcome at this time.

However, if the site is blocked then first-party domain blocking is on by default. This should not be the case for Release Channel builds at this time.

**Comment by LaurenWags** [comment]
@brave/legacy_qa we will need a 1.23.x RC to check this issue.

**Comment by kjozwiak** [comment]
STR/Cases to run through:

* _**Verify domain blocking is `disabled` on release channel**_ - visit pub.chez.com and make sure there isn't a `Suspicious site ahead` interstitial screen being loaded/displayed
* _**Verify domain blocking can be `enabled` on release channel**_ - go to `brave://flags`, set `#brave-domain-block` to `Enabled`, visit pub.chez.com, and make sure that there is a `Suspicious site ahead` interstitial screen being loaded/displayed
* Repeat the above using either Nightly/Dev/BETA and you make sure having  `#brave-domain-block` as Default causes the `Suspicious site ahead` interstitial to appear

**Comment by LaurenWags** [comment]
Removing `QA/Blocked` since https://github.com/brave/brave-browser/releases/tag/v1.23.68 is available

**Comment by kjozwiak** [comment]
Waiting on @iefremov to push these changes into Griffen.

**Comment by kjozwiak** [comment]
Worked with @iefremov and went through the following on `Staging` and ensured `Top Level Domain Blocking` was disabled:

### `1.23.x Test Case`

Launched `1.23.69 CR: 90.0.4430.61` using the following:
```
brave.exe --enable-logging=stderr --variations-server-url=https://variations.bravesoftware.com/seed
```

**Note:** You'll need to restart the browser after the initial launch using the same switch. Once relaunched, checked `brave://version/?show-variations-cmd` and ensured the following:

![domainBlocking](https://user-images.githubusercontent.com/2602313/114233361-2d81a480-994b-11eb-82ca-50acea58d907.png)

Check `brave://local-state/` and ensure that you see data re: the variation server/seed as per the following:

```
   },
   "variations_compressed_seed": "safe_seed_content",
   "variations_country": "CA",
   "variations_crash_streak": 0,
   "variations_failed_to_fetch_seed_streak": 0,
   "variations_last_fetch_time": "13262471257436368",
   "variations_permanent_consistency_country": [ "90.1.23.69", "CA" ],
   "variations_safe_compressed_seed": "H4sIAAAAAAAAAL3OPQ/BQACA4TskOEtzSUUsGtPFcOmXatemFiN/wF3vSqMfUiWxEv9bG4MUs3d+hwdp3DTn3DVtx+BV0nEcL1qEOg+51EPLZvgGkeoX7CKDPGVx5id5eNiUZ3F14WqCusuM8UQKBXCMlM9vpaFeEJ9eh+AYfx8D1A1kxM5JqYC1Ou57OjWoadGZ1iaAQNIinS3E9wqxPO5lKguWbMq8YDv5AyFqxOfXQIAa8XU0EEOMXK9GGHRhNRSPPypGDQXQWm/HFBpPuM9RyLgBAAA=",
   "variations_safe_seed_date": "13262471142000000",
   "variations_safe_seed_fetch_time": "13262471257436368",
   "variations_safe_seed_locale": "en-GB",
   "variations_safe_seed_permanent_consistency_country": "CA",
   "variations_safe_seed_session_consistency_country": "CA",
   "variations_safe_seed_signature": "MEQCIAOkK3lKIetQ3piryhqQox29I1N501knG8pcpXbic695AiAK7lwK1IP07f/mrLZXdo8o3wjblJyZ7pWeMNnpUsy5gQ==",
   "variations_seed_date": "13262471257000000",
   "variations_seed_signature": "MEQCIAOkK3lKIetQ3piryhqQox29I1N501knG8pcpXbic695AiAK7lwK1IP07f/mrLZXdo8o3wjblJyZ7pWeMNnpUsy5gQ==",
   "was": {
      "restarted": false
   }
```

Once the above has been confirmed, checked the following websites and ensured that the interstitials page wasn't being displayed.

* `1-1ads.com` - **PASSED**
* `actionsplash.com` - **PASSED**
* `pub.chez.com` - **PASSED**

### `1.25.x Test Case`

Launched `1.25.4 CR: 90.0.4430.51` using the following:
```
brave.exe --enable-logging=stderr --variations-server-url=https://variations.bravesoftware.com/seed --fake-variations-channel=stable
```

![domainBlocking](https://user-images.githubusercontent.com/2602313/114234490-d381de80-994c-11eb-85db-db2fd7fef858.png)

Once the above has been confirmed, checked the following websites and ensured that the interstitials page was being displayed.

* `1-1ads.com` - **PASSED**
* `actionsplash.com` - **PASSED**
* `pub.chez.com` - **PASSED**

**Comment by kjozwiak** [comment]
Unblocking as https://github.com/brave/brave-variations/pull/14 has been merged into production. There's still one issue re: new installs needing to restart the browser. Still TBD if this will get addressed by `1.23.x` or later.

**Comment by srirambv** [comment]
Verification passed on OnePlus 6T with Android 10 running 1.23.69 x64 build
- Verified test plan from https://github.com/brave/brave-browser/issues/15149#issuecomment-816936276
- Verified variations are shown in `brave://version/?show-variations-cmd` 
- Verified websites like `1-1ads.com` & `actionsplash.com` don't show interstitial page when loaded 
- Verified without variations the above pages show interstitial page
- Verified `brave://local-state` shows variations log

```json
   },
   "variations_compressed_seed": "H4sIAAAAAAAAAL3OPQ/BQACA4TskOEtzSUUsGtPFcOmXatemFiN/wF3vSqMfUiWxEv9bG4MUs3d+hwdp3DTn3DVtx+BV0nEcL1qEOg+51EPLZvgGkeoX7CKDPGVx5id5eNiUZ3F14WqCusuM8UQKBXCMlM9vpaFeEJ9eh+AYfx8D1A1kxM5JqYC1Ou57OjWoadGZ1iaAQNIinS3E9wqxPO5lKguWbMq8YDv5AyFqxOfXQIAa8XU0EEOMXK9GGHRhNRSPPypGDQXQWm/HFBpPuM9RyLgBAAA=",
   "variations_country": "IN",
   "variations_crash_streak": 0,
   "variations_failed_to_fetch_seed_streak": 0,
   "variations_last_fetch_time": "13262696710793465",
   "variations_permanent_consistency_country": [ "90.1.23.69", "IN" ],
   "variations_safe_compressed_seed": "H4sIAAAAAAAAAL3OPQuCQACA4TsLrGuRgyJakiZpEO/KrzV0caw/0OmdFVmGadBa9L9TGkJtbni3d3iQG5scOMTklBluavMyKljy2KDVsS4QRdfEDouEqYzfhpSd2OK+SNDpu8oLfHRhMkeyfWZgIroAQI6X5BSrqeYfr5+Ahxu1jgGRPxKxIcgWsh5O+a+hEpwt9rnY0oEFN0rpbiJ8lwr/sxUlkLNnkacZ24geCV4jmV0OACtE6aogRRo5bIYhuL2qK1x8V45oCqNLXMYPkDcntuxi4AQAA",
   "variations_safe_seed_date": "13262694873000000",
   "variations_safe_seed_fetch_time": "13262694876299507",
   "variations_safe_seed_locale": "en-US",
   "variations_safe_seed_permanent_consistency_country": "IN",
   "variations_safe_seed_session_consistency_country": "IN",
   "variations_safe_seed_signature": "MEUCIQD0oqynrJRGgJNLQUhz/SzgdVJNRw8WU7YmNgaS5x6BOAIgHetkiltEjq17sF/0+aTx2wnCssKhnuCCImSSWrs5nTU=",
   "variations_seed_date": "13262696711000000",
   "variations_seed_signature": "MEQCIAOkK3lKIetQ3piryhqQox29I1N501knG8pcpXbic695AiAK7lwK1IP07f/mrLZXdo8o3wjblJyZ7pWeMNnpUsy5gQ=="
}
```

![image](https://user-images.githubusercontent.com/17010094/114384139-4033ed80-9b86-11eb-9fe5-466d4a90950d.png) | ![image](https://user-images.githubusercontent.com/17010094/114384216-56da4480-9b86-11eb-9ebe-4ee5f3ef7651.png)
-- | --

-----------

Verification passed on Samsung Tab A with Android 10 running 1.23.69 x64 build
- Verified test plan from https://github.com/brave/brave-browser/issues/15149#issuecomment-816936276
- Verified variations are shown in `brave://version/?show-variations-cmd` 
- Verified websites like `1-1ads.com` & `actionsplash.com` don't show interstitial page when loaded 
- Verified without variations the above pages show interstitial page
- Verified `brave://local-state` shows variations log

```json
},
   "variations_compressed_seed": "H4sIAAAAAAAAAL3OPQ/BQACA4TskOEtzSUUsGtPFcOmXatemFiN/wF3vSqMfUiWxEv9bG4MUs3d+hwdp3DTn3DVtx+BV0nEcL1qEOg+51EPLZvgGkeoX7CKDPGVx5id5eNiUZ3F14WqCusuM8UQKBXCMlM9vpaFeEJ9eh+AYfx8D1A1kxM5JqYC1Ou57OjWoadGZ1iaAQNIinS3E9wqxPO5lKguWbMq8YDv5AyFqxOfXQIAa8XU0EEOMXK9GGHRhNRSPPypGDQXQWm/HFBpPuM9RyLgBAAA=",
   "variations_country": "IN",
   "variations_crash_streak": 0,
   "variations_failed_to_fetch_seed_streak": 0,
   "variations_last_fetch_time": "13262694348918483",
   "variations_permanent_consistency_country": [ "90.1.23.69", "IN" ],
   "variations_safe_compressed_seed": "H4sIAAAAAAAAAL3OPQuCQACA4TsLrGuRgyJakiZpEO/KrzV0caw/0OmdFVmGadBa9L9TGkJtbni3d3iQGscOMTklBluavMyKljy2KDVsS4QRdfEDouEqYzfhpSd2OK+SNDpu8oLfHRhMkeyfWZgIroAQI6X5BSrqeYfr5+Ahxu1jgGRPxKxIcgWsh5O+a+hEpwt9rnY0oEFN0rpbiJ8lwr/sxUlkLNnkacZ24geCV4jmV0OACtE6aogRRo5bIYhuL2qK1x8V45oCqNLXMYPkDcntuxi4AQAA",
   "variations_safe_seed_date": "13262688324000000",
   "variations_safe_seed_fetch_time": "13262688352684664",
   "variations_safe_seed_locale": "en-US",
   "variations_safe_seed_permanent_consistency_country": "IN",
   "variations_safe_seed_session_consistency_country": "IN",
   "variations_safe_seed_signature": "MEUCIQD0oqynrJRGgJNLQUhz/SzgdVJNRw8WU7YmNgaS5x6BOAIgHetkiltEjq17sF/0+aTx2wnCssKhnuCCImSSWrs5nTU=",
   "variations_seed_date": "13262694323000000",
   "variations_seed_signature": "MEQCIAOkK3lKIetQ3piryhqQox29I1N501knG8pcpXbic695AiAK7lwK1IP07f/mrLZXdo8o3wjblJyZ7pWeMNnpUsy5gQ=="
}
```

<img width=400 src=https://user-images.githubusercontent.com/17010094/114385315-c997ef80-9b87-11eb-8e10-08987dcaf27a.png>

**Comment by btlechowski** [comment]
Verification passed on

Brave | 1.23.69 Chromium: 90.0.4430.61 (Official Build) (64-bit)
-- | --
Revision | dced74d4124b26b14126b611853d33512b60c7b6-refs/branch-heads/4430@{#1115}
OS | Ubuntu 18.04 LTS

Verified test plan from https://github.com/brave/brave-browser/issues/15149#issuecomment-816936276 on Brave release and variation seed production server
![image](https://user-images.githubusercontent.com/34715963/114408547-ce6d9b00-9ba9-11eb-92c6-731f3b557a4a.png)

Verified that the interstitials page wasn't being displayed
* `1-1ads.com` - **PASSED** - page was unreachable
* `actionsplash.com` - **PASSED**
* `pub.chez.com` - **PASSED**

**Comment by kjozwiak** [comment]
Closing as @pilgrim-brave fixed/merged https://github.com/brave/brave-core/pull/8507 into `1.23.x`. We'll need to retest the above once we get a new `1.23.x` RC.

**Comment by LaurenWags** [comment]
Adding `QA/Blocked` until a new RC is built per https://github.com/brave/brave-browser/issues/15149#issuecomment-818383487

**Comment by stephendonner** [comment]
Removing `QA/Blocked` as we have a new `1.23.70` build.

Verified `PASSED` using 

Brave | 1.23.70 Chromium: 90.0.4430.70 (Official Build) (x86_64)
-- | --
Revision | 3954de7175366f3b7edca576f140dfa273e6b5ae-refs/branch-heads/4430@{#1210}
OS | macOS Version 11.2.3 (Build 20D91)

Tested and ensured that 1st-party domain blocking is *OFF* by successfully loading:

actionsplash.com | adap.tv | pub.chez.com
------------------|---------|---------------
<img width="959" alt="Screen Shot 2021-04-13 at 3 25 53 PM" src="https://user-images.githubusercontent.com/387249/114629085-d0e4f900-9c6c-11eb-8d70-492e73a4637c.png"> | <img width="959" alt="Screen Shot 2021-04-13 at 3 26 26 PM" src="https://user-images.githubusercontent.com/387249/114629087-d17d8f80-9c6c-11eb-8817-e8c370142bfa.png"> | <img width="959" alt="Screen Shot 2021-04-13 at 3 26 06 PM" src="https://user-images.githubusercontent.com/387249/114629090-d17d8f80-9c6c-11eb-990c-490e4b5e8708.png">

**Comment by srirambv** [comment]
Verification passed on OnePlus 6T with Android 10 running 1.23.70
- Verified domain block is set to default OFF
- Verified visiting the test domains doesn't show safe browsing interstitial page

![image](https://user-images.githubusercontent.com/17010094/114661317-b5223700-9cee-11eb-8664-59cc4b30463c.png) | ![image](https://user-images.githubusercontent.com/17010094/114661370-c9663400-9cee-11eb-82bc-f6038268a8a9.png) | ![image](https://user-images.githubusercontent.com/17010094/114661379-ccf9bb00-9cee-11eb-9354-1d6112b13e12.png)
-- | -- | --
--------
Verification passed on Samsung Tab A with Android 10 running 1.23.70
- Verified domain block is set to default OFF
- Verified visiting the test domains doesn't show safe browsing interstitial page

![image](https://user-images.githubusercontent.com/17010094/114661162-742a2280-9cee-11eb-9ef3-3d6da13f3071.png) | ![image](https://user-images.githubusercontent.com/17010094/114661168-79876d00-9cee-11eb-9207-1ebdf2f494ed.png)  | ![image](https://user-images.githubusercontent.com/17010094/114661177-7c825d80-9cee-11eb-8c83-a54e9759f982.png)
-- | -- | --
